app/src/main/cpp/ControllerContainer.cpp

```diff
@@ -270,6 +275,25 @@ void ControllerContainer::SetHandJointLocations(const int32_t aControllerIndex,
         assert(controller.handJointTransforms[i]);
         controller.handJointTransforms[i]->SetTransform(transform);
     }
+#else
+    std::vector<vrb::Matrix> tmpJointTransforms = jointTransforms;
+    // We ignore the first matrix, corresponding to the palm, because the
+    // models we are currently using don't include a palm joint.
+    tmpJointTransforms.erase(tmpJointTransforms.begin());
+
+    if (controller.meshJointTransforms.size() == 0)
+        controller.meshJointTransforms.resize(tmpJointTransforms.size());
+    else
+        assert(controller.meshJointTransforms.size() == tmpJointTransforms.size());
```

Comment:
Just place the assert after the if and not in the else branch, because it should be true even if the if branch was taken.

Reply:
Right, will fix that.